apps/dashboard/src/components/primitives/form/avatar-picker.tsx

```diff
@@ -25,25 +31,31 @@ const predefinedAvatars = [
   `${window.location.origin}/images/error-warning.svg`,
 ];
 
-type AvatarPickerProps = React.InputHTMLAttributes<HTMLInputElement>;
+type AvatarPickerProps = {
+  name: string;
+  value: string;
+  onChange?: (value: string) => void;
+};
 
-export const AvatarPicker = forwardRef<HTMLInputElement, AvatarPickerProps>(({ id, ...props }, ref) => {
+export const AvatarPicker = forwardRef<HTMLInputElement, AvatarPickerProps>(({ name, value, onChange }, ref) => {
+  const { step } = useStepEditorContext();
+  const variables = useMemo(() => (step ? parseStepVariablesToLiquidVariables(step.variables) : []), [step]);
   const [isOpen, setIsOpen] = useState(false);
   const { error } = useFormField();
 
   const handlePredefinedAvatarClick = (url: string) => {
-    props.onChange?.({ target: { value: url } } as React.ChangeEvent<HTMLInputElement>);
+    onChange?.(url);
     setIsOpen(false);
   };
 
   return (
-    <div className="space-y-2">
+    <div className="size-10 space-y-2">
       <Popover modal={true} open={isOpen} onOpenChange={setIsOpen}>
         <PopoverTrigger asChild>
           <Button variant="outline" size="icon" className="text-foreground-600 relative size-10">
-            {props.value ? (
+            {value ? (
               <Avatar className="p-px">
-                <AvatarImage src={props.value as string} />
+                <AvatarImage src={value as string} />
```

Comment:
```suggestion
                <AvatarImage src={value} />
```
It's a string already so this isn't needed